src/main/resources/static/index.html

```diff
@@ -10,5 +10,5 @@
 </head>
 <body>
     <p>Single line showed below.</p>
-    <img src="/api/image">
```

Comment:
what's purpose of this change?

Reply:
I think I made a mistake there, the correct path is in new version